ansible/roles/grafana/templates/grafana_nginx.conf.j2

```diff
@@ -0,0 +1,20 @@
+server {
```

Comment:
I usually write a comment like `# ansible-managed in ooni-sysadmin.git` in every managed file so it's obvious that it should not be edited manually.
I was inspired by lots of file generators adding `DO NOT EDIT! File autogenerated by blah-blah.` comment :)